Blear/SSAppDelegate.m

```diff
@@ -23,15 +23,17 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 	if ([[NSUserDefaults standardUserDefaults] valueForKey:@"firstTime"] == NULL) {
 		[[NSUserDefaults standardUserDefaults] setValue:@"not" forKey:@"firstTime"];
 
-		// TODO: `UIAlertView` is deprecated. Switch it out with `UIAlertController`.
-		[[
-		  [UIAlertView alloc]
-		  initWithTitle:@"Tip"
-		  message:@"Shake the device to get a random image."
-		  delegate:self
-		  cancelButtonTitle:@"Cool"
-		  otherButtonTitles:nil
-		  ] show];
+        UIAlertController * alert = [UIAlertController alertControllerWithTitle:@"Tip"
+                                                                        message:@"Shake the device to get a random image."
+                                                                 preferredStyle:UIAlertControllerStyleAlert];
+        
+        UIAlertAction* coolAction = [UIAlertAction actionWithTitle:@"Cool"
+                                                             style:UIAlertActionStyleDefault
+                                                           handler:nil];
+        
+        [alert addAction:coolAction];
+        
+        [self.inputViewController presentViewController:alert animated:YES completion:nil];
```

Comment:
`self.inputViewController` => `rootViewController`

It currently doesn't work with `self.inputViewController`